config/webpack.config.base.js

```diff
@@ -41,6 +41,7 @@ module.exports = {
     ]
   },
   plugins: [
+    new webpack.ContextReplacementPlugin(/date-fns[\/\\]locale$/, /en/),
```

Comment:
And for the other languages we will load them asynchronously ? Why does the bar need date-fns ?

Reply:
La cozy-bar n'a actuellement pas besoin de date-fns mais la librairie est importé puisque l'on utilise I18n de cozy-ui...

Reply:
Ah oui OK. C'est surement plus simple alors de faire `const I18n = require('cozy-ui/react/I18n')` non ? En fait je pense que dans les librairies on devrait pas utiliser le `require('cozy-ui/react')`... Désolé :/ ...

Reply:
OK compris, c'est bon c'est I18n qui importe date-fns, désolé. LGTM !